src/todos/interfaces/todo.interface.ts

```diff
@@ -0,0 +1,7 @@
+export interface Todo {
+    id?: string;
```

Comment:
Why is the id optional? 
Also -  doesn't Mongo give a _id property?

Reply:
1. Because we are not always going to include the id, because it's included by the DB automatically. 
2. Yes it does! 
I did this part with